documentation/en/user/source/configuration/storage.rst

```diff
@@ -272,6 +271,173 @@ GeoServer ``topp:states`` sample layer on a fictitious ``my-geowebcache-bucket``
+
+Follows an example of what the blob store root directory structure may look like when using the default file path template:
+
+.. code-block:: none
+
+  .
+  `-- EPSG_4326
+      `-- sf_restricted
+          `-- image_png
+              |-- f0023dc7bc347fee7a3a04dc797f2223f74e3448
+              |   `-- 14
+              |       `-- tiles_6750_12000.sqlite
+              `-- null
+                  |-- 10
+                  |   |-- tiles_0_0.sqlite
+                  |   |-- tiles_250_500.sqlite
+                  |   `-- tiles_250_750.sqlite
+                  `-- 9
+                      |-- tiles_0_0.sqlite
+                      `-- tiles_0_250.sqlite
+
+If no parameters were provided *null* string will be used. Is the responsibility of the user to define a file path template that will avoid collisions.
+
+The terms that can be used in the file path template are:
+
+* **grid**: the grid set id
+* **layer**: the name of the layer
+* **format**: the image format of the tiles
+* **params**: parameters unique hash
+* **x**: column range, computed based on the column range count configuration property
+* **y**: row range, computed based on the row range count configuration property
+* **z**: the zoom level
+
+It is also possible to use parameters values, like *style* for example. If the parameter is not present *null* will be used.
+
+.. note:: The file template separators are specific to the operating system used, for example ``\`` for Linux and ``/`` for Windows. Any special char like \, /, : or empty space used in a term value will be substituted with an underscore.
+
+MBTiles Metadata
+`````````````````
+
+A valid MBTiles file will need some metadata, the image format and layer name will be automatically added when an MBTiles file is created. The user can provide the remaining metadata using a properties file whose name must follow this pattern:
+
+.. code-block:: none
+
+  <layerName>.metadata
+
+The directory that contains this metadata files is defined by a configuration property.
+
+Expiration Rules
+`````````````````
+
+The MBTiles specification don't give us a way to know when a tile was created. To allow expire rules we use an auxiliary table were we will store that information. In the presence of an MBTiles file generated by a third party tool we will assume that the create time of a tile was the first time it was accessed. This feature can be activated or deactivated by configuration. Note that this will not break the MBTiles specification compliance.
+
+Eager Truncate
+```````````````
+
+When performing a truncate of the cache we should always try to remove the whole database file avoiding to create fragmented space. This is not suitable for all the situations and is highly dependent on the database files granularity. The configuration property ``eagerDelete`` allows us to disable or deactivate this feature which is disabled by default.
```

Comment:
Explain under which conditions the whole file is deleted. Like, if one issues a truncate for a given bbox or a given set of zoom levels, will this wipe out the whole database if eager delete is set? Or does the code has a smart way of knowing if a filter matches the entire contents of a database?